API-Samples/Java/app/src/main/java/eventprocessor/VideoIndexerEventProcessor.java

```diff
@@ -0,0 +1,65 @@
+package eventprocessor;
+
+import com.azure.identity.*;
+import com.azure.messaging.eventhubs.EventProcessorClient;
+import com.azure.messaging.eventhubs.EventProcessorClientBuilder;
+
+import java.util.concurrent.Semaphore;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * The EventProcessor Sample is take from <a href="https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/eventhubs/azure-messaging-eventhubs/README.md#consume-events-using-an-eventprocessorclient">...</a>
+ * For more information about Azure Event Hubs SDK For Java visit : <a href="https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/eventhubs/azure-messaging-eventhubs/README.md#azure-event-hubs-client-library-for-java">...</a>
+ */
+public class VideoIndexerEventProcessor {
+    // The fully qualified namespace for the Event Hubs instance. This is likely to be similar to:
+    // {your-namespace}.servicebus.windows.net
+    private static final String eventHubsFqns = "<Your Event Hubs Full Qualified Namespace here>";

```

Comment:
Consider using environment variables for fqns, eventHubName, and consumerGroupName so it will be easy to run locally.

Reply:
next iteration, I promise